
{'date': 'Wed Aug 22 2018 15:32:43.922 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 20
  falatarzan = Texto(floresta, "Que fome !!!)
                                             ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Wed Aug 22 2018 15:33:05.572 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 20
  falatarzan = Texto(floresta, "Que fome !!!)
                                             ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Wed Aug 22 2018 15:34:23.350 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 20
  falatarzan = Texto(floresta, "Que fome !!!)
                                             ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Wed Aug 22 2018 15:52:12.346 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 34
    criarcenas()
  module <module> line 30
    alice.entra (mesa)
NameError: name 'alice' is not defined
'''},
{'date': 'Wed Aug 22 2018 16:03:24.246 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    STYLE ["width"]=900
NameError: name 'STYLE' is not defined
'''},
{'date': 'Wed Aug 22 2018 16:03:35.120 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    STYLE["width"]=900
NameError: name 'STYLE' is not defined
'''},